source/views/streaming/radio/schedule.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+
+import React from 'react'
+import {GoogleCalendarView} from '../../calendar/calendar-google'
+
+export default class KSTOSchedule extends React.PureComponent {
+  static navigationOptions = {
+    title: 'Show Schedule',
+  }
+
+  render() {
+    return (
+      <GoogleCalendarView
```

Comment:
We need a way  to provide a customized `powered by` message to the calendar. This is now a known-issue on `master`.

Reply:
@hawkrives fixed this in #1904